docs/docs/icicle/golang-bindings/msm.md

```diff
@@ -160,41 +162,42 @@ out.Malloc(batchSize*p.Size(), p.Size())
 To activate G2 support first you must make sure you are building the static libraries with G2 feature enabled.
 
 ```bash
-./build.sh bls12_381 ON
+./build.sh bn254 -g2
```

Comment:
I think we should only have a single place to show how to build with different features. Otherwise they will get lost in the docs and never be up to date

Reply:
If we want to reference that single location in many places, thats fine. but it doesn't make sense to repeat the same thing

Reply:
Agree